hail/src/main/scala/is/hail/backend/spark/SparkBackend.scala

```diff
@@ -22,13 +23,35 @@ object SparkBackend {
     Serialization.write(Map("value" -> jsonValue, "timings" -> timings.value))(new DefaultFormats {})
   }
 
+  def jvmExecute(sc: SparkContext, ir0: IR, optimize: Boolean = true): (Any, Timings) = {
```

Comment:
Why is this piece added?

Reply:
Why can't we just run CompileAndEvaluate?

Reply:
CompileAndEvaluate doesn't lower. (This path lowers and runs the lowered pipeline through the JVM--it's basically the equivalent of what the c++ execution path was going to be.)

Reply:
It also guarantees that we're not running anything through the interpreter, which CompileAndEvaluate doesn't do.

Reply:
ok, got it. I sorta figured this out after commenting, but thanks for confirming! Can we call it something that indicates it lowers table IRs? Then good to go.

Reply:
done!